dashboards/iis/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+category: IIS
```

Comment:
I'll note that the format for metadata.yaml is still being finalized so there's expectation for it to be included as of now. Feel free to leave or revert, I'll make any necessary adjustments later and outline the expected format in the contribution docs when it's ready.

Reply:
gotcha, reverted for now then.